spec/Section 5 -- Validation.md

```diff
@@ -1358,6 +1359,31 @@ For example the following query will not pass validation.
 ```
 
 
+### Input Object Required Fields
```

Comment:
This rule exists as part of GraphQL.js's `ValuesOfCorrectType` but was missing from the spec